web-console/4.0-designs/global-config/global-config.md

```diff
@@ -0,0 +1,25 @@
+# Global Config
+
+![1](img/D-01B.png)
+
+- Global Config is the third tab of the [Cluster Settings](http://openshift.github.io/openshift-origin-design/web-console/4.0-designs/cluster-settings/cluster-settings) page.
```

Comment:
@beanh66 I'm not sure we should be using an abbreviation here, as we typically try to stay away from that in the UI.

Reply:
@serenamarie125 that came from a comment from Jessica. I originally had Global Configuration but seemed to be an issue for people. Happy to change it back and I agree with you it may be odd for us to abbreviate. Note in Robb's current implementation the tab is called Global Config.